statsrelay.go

```diff
@@ -142,7 +169,7 @@ func sendPacket(buff []byte, target string, sendproto string, TCPtimeout time.Du
 			break
 		}
 		conn.Write(buff)
-		conn.Close()
+		defer conn.Close()
```

Comment:
Yes, we need some overall better testing and right now i do some tests on higher traffic client most with flamegraphs and pprof to look what takes most of CPU and this is only little change to make it better before bigger changes.
Main problem in statsrelay right now is dial on TCP. 
We are making new connection sending small packet (mtu defined size) and then closing connection. Moving closing to go-routine should help, but final solution is to build buffering and sending data on one connection in configured batches of packets and then reconnect. 
We need buffers/or structure per hashring endpoint or rehashing if enpoint return time expire.
This makes possible to have fail buffer when connection i/o errors and store data until hash ring endpoint return with some buffer limits and remove last old when no space in buffer.

I would like to implement this but in near future because now i need to focus on different part of monitoring infrastructure.